django/santropolFeast/page/views.py

```diff
@@ -8,12 +8,14 @@
 from django.core.urlresolvers import reverse_lazy
 from django.utils.decorators import method_decorator
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from member.models import Profile
 
 
 @login_required
 def home(request):
-    return render(request, 'pages/home.html')
+    profile = get_object_or_404(Profile, user=request.user)
```

Comment:
It will be nice to add a security on login view to auth user only if he have a profile.